pipelines/rj_smtr/veiculo/tasks.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+"""
+Tasks for veiculos
+"""
+
+from datetime import timedelta
+import traceback
+import pandas as pd
+from prefect import task
+
+# EMD Imports #
+
+from pipelines.utils.utils import get_vault_secret, log
+
+# SMTR Imports #
+
+from pipelines.rj_smtr.constants import constants
+from pipelines.rj_smtr.utils import log_critical, map_dict_keys
+
+# Tasks #
+
+
+@task
+def pre_treatment_veiculos(status: dict, timestamp: str):
```

Comment:
```suggestion
def pre_treatment_veiculo_sppo_licenciamento(status: dict, timestamp: str):
```

é especifico para o licencimento do sppo, e nao para outras tabelas que podem surgir no dataset. mudar as referencias tbm.